### ISSUE TITLE
Video Quality 

### ISSUE THREAD

**Issue by Mezar303** [JinRonin/plugin.video.eurosportplayer#3]
I would like to choose batter video quality than the only one possible 960x540. Can you add this functionality?

**Comment by JinRonin** [comment]
Just check out the eurosport player addon settings -> videoplayer.

**Comment by raintonr** [comment]
I looked in the settings and found min/max bitrates plus some resolution settings.

By default the min/max are zero (I would assume that means no limit) and resolution were at max. Setting the min/max bitrate to anything else (I tried 5000min 10000max) yields a substantial drop in quality.

Why I looked at this is the following: we have a Sony smart TV (XE9005) and think the picture quality using Kodi plus this plugin is not as good as watching on a laptop then using Chrome's 'cast to screen' function. Which seems odd to me. There is also an issue whereby the stream is much more stable decoding on a laptop and casting. Our network isn't great, but it's plenty for Eurosport player.

Any ideas what's going on here? Any clarification on what the settings -> videoplayer attributes should actually do?

Thanks :)
